armi/settings/fwSettings/globalSettings.py

```diff
@@ -661,6 +662,14 @@ def defineSettings() -> List[setting.Setting]:
             description="Blocks with these flags will not move in moves. "
             "Used for fuel management.",
         ),
+        setting.Setting(
+            CONF_STATIONARY_BLOCKS,
+            default=[],
+            label="stationary Blocks",
+            description="Blocks with these indices (int values) will not move in "
+            "moves. Used for fuel management. "
+            "Deprecated setting, use CONF_STATIONARY_BLOCK_FLAGS",
+        ),
```

Comment:
Why is this here?

1. It doesn't appear to be used.
2. It already says it's deprecated and we shouldn't use it?

Reply:
Not sure how this got  back in. I think I probably made a mistake when merging.